src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -185,4 +186,20 @@ public static MembershipPoints parseMembershipPoints(String membershipPoints) th
         }
         return new MembershipPoints(trimmedMembershipPoints);
     }
+
+    /**
+     * Parses a {@code String itemName} into an {@code itemName}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code itemName} is invalid.
```

Comment:
Since itemName is not a class i dont think we should use the  @code label for it. I think a better javadoc comment would be: "processes the given string into the proper format for an Item's Name"